platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -103,22 +103,6 @@ - (BOOL)intersectsOverlayBounds:(MGLCoordinateBounds)overlayBounds
     return MGLLatLngBoundsFromCoordinateBounds(_bounds).intersects(MGLLatLngBoundsFromCoordinateBounds(overlayBounds));
 }
 
-- (mbgl::AnnotationSegments)annotationSegments {
-    NSUInteger count = self.pointCount;
-    CLLocationCoordinate2D *coordinates = self.coordinates;
-    
-    mbgl::AnnotationSegment segment;
-    segment.reserve(count);
-    for (NSUInteger i = 0; i < count; i++) {
-        segment.push_back(MGLLatLngFromLocationCoordinate2D(coordinates[i]));
-    }
-    return { segment };
-}
-
-- (mbgl::ShapeAnnotation::Properties)shapeAnnotationPropertiesObjectWithDelegate:(__unused id <MGLMultiPointDelegate>)delegate {
```

Comment:
Agreed. Unfortunately, I think we’re stuck with [MapKit’s concept of a multipoint](https://developer.apple.com/library/mac/documentation/MapKit/Reference/MKMultiPoint_class/) (as any shape with multiple points) for backwards compatibility. What could we call the class that represents a true multipoint? Or should we bite the bullet and bump the major version number for simplicity?